flink-connectors/flink-sql-connector-hive-1.2.2/src/main/resources/META-INF/licenses/LICENSE.json

```diff
@@ -0,0 +1,9 @@
+Copyright (c) 2002 JSON.org
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+The Software shall be used for Good, not Evil.
```

Comment:
This sentence is the problem (it makes the licenses non-free): https://wiki.debian.org/qa.debian.org/jsonevil

Reply:
Yes, this is a category-X license. If this is required for Hive, then Hive has a big problem, we we will not be able to depend on hive-exec.

Reply:
@zentol  Do you mean we're not allowed to depend on hive-exec or we're not allowed to bundle hive-exec?

Reply:
Let's continue this discussion in  https://github.com/apache/flink/pull/12682#discussion_r440880126.